### ISSUE TITLE
Making a note/announcment on calendar

### ISSUE THREAD

**Issue by AliCihan** [pitaj/nodebb-plugin-calendar#125]
Can you please add another functionality, instead of creating an event, it will be creating notes/announcements on the calendar for whole forum to see...  So, it is important information, but people do not need to attend... 

such as... "NodeBB is now available as Digital Ocean one click app"

And it may have direct link to forum topic or any other website link...

As mentioned here:
https://community.nodebb.org/topic/4401/nodebb-plugin-calendar-fully-featured-calendar-plugin-for-nodebb-testers-needed/439
> is it possible to make a note on calendar? It will not be an event where people go, it will just mark an important day and have a description. (For example, each month we have "visa bulletin" announced by DOS, it would be nice to add that on calendar and give direct link to the bulletin rather than a topic in the forum). In this case, of course it will have more simple look because people does not need to say yes/no/maybe...

**Comment by pitaj** [comment]
Would this be accomplished by the ability to disable responses on a per-event basis?

**Comment by AliCihan** [comment]
If there is a related topic on the forum, yes... This is how we are doing it right now for most of the cases... But there are cases where we do not need a separate topic for the announcement, or not topic at all... so, it would be nice to add an announcement events on the calendar page , rather than forming an event on the topic.

**Comment by pitaj** [comment]
You can create events without creating a separate topic. You can create events in posts.

**Comment by pitaj** [comment]
I guess I'm just confused in how you want these notes to work. How are they created? How are they interacted with? Who can create them? What information is associated with them?

**Comment by AliCihan** [comment]
Yes, what I meant is that announcement is not necessarily related to any topic on the forum, but it is for the interest of forum users.

**Comment by AliCihan** [comment]
Just like how it works on our regular calendars. For example you just say "Add Note" on the calendar page, and it creates a note for everyone to see (maybe you can limit the audience)... 

For our purpose, it is enough for moderators/admins to create a note like that. But for others, it might be useful if regular users also make a note/announcement on calendar.

**Comment by pitaj** [comment]
Apologies for taking so long to respond, I do this in my free time. Is the note associated with a specific date/time?

**Comment by AliCihan** [comment]
no problem, thanks for your time. 

It is a specific date or date range since it will appear on a particular place on the calendar. But it does not have a particular time in our case.
